pkg/storage/enginepb/mvcc.proto

```diff
@@ -185,3 +185,72 @@ message MVCCStats {
   // abort span. These bytes are a subset of sys_bytes.
   optional sfixed64 abort_span_bytes = 15 [(gogoproto.nullable) = false];
 }
+
+// MVCCStatsLegacyRepresentation is almost identical to MVCCStats, except
+// it does not have a field tracking abort span bytes.
+// The abort span bytes field in MVCCStats didn't exist in earlier versions,
+// and its addition of it to MVCCStats causes the struct to be larger than it was
+// previously. The discrepancy between the size of MVCCStats between versions
+// could cause incorrect sysbyte counts during version migration.
+// MVCCStatsLegacyRepresentation is the older version of MVCCStats, and allows
+// us to prevent this discrepancy during version migration.
+message MVCCStatsLegacyRepresentation {
+  option (gogoproto.equal) = true;
+  option (gogoproto.populate) = true;
+
+  // contains_estimates indicates that the MVCCStats object contains values
```

Comment:
Let's strip out all these comments because we don't actually expect folks to be using this struct.